lxc/exec_unix.go

```diff
@@ -20,7 +20,7 @@ func (c *execCmd) getStdout() io.WriteCloser {
 
 func (c *execCmd) controlSocketHandler(d *lxd.Client, control *websocket.Conn) {
 	ch := make(chan os.Signal)
-	signal.Notify(ch, syscall.SIGWINCH, syscall.SIGTERM)
+	signal.Notify(ch, syscall.SIGWINCH, syscall.SIGTERM, syscall.SIGHUP)
```

Comment:
Commit should be squashed with previous one since all the SIGHUP handling is in there already.